contributors/index.js

```diff
@@ -67,6 +67,14 @@ module.exports.list = [
     github_username: "RealBigpunch",
     twitter_username: "MeisterShill"
   },
+  {
+    name: "Sibin Joseph",
+    country: "IN",
+    role: "To be filled",
```

Comment:
Can you fill in the role property with what type of developer you are or want to be ? (ex : Frontend developer)